AppCenter/AppCenter/Internals/Model/MSAbstractLog.m

```diff
@@ -22,7 +22,7 @@ @implementation MSAbstractLog
 - (instancetype)init {
   self = [super init];
   if (self) {
-    self.tenants = [NSMutableSet new];
+    self.transmissionTargetTokens = [NSSet new];
```

Comment:
The reason is that because we are not using `getTransmissionTargetTokens` but `transmissionTargetTokens`, the compiler now complains about the type mismatch.
I was thinking that we don't want the property mutable so that's why.
Does that make sense?

Reply:
I see it's now a classic getter on the property, it's fine.